scripts/portstat

```diff
@@ -48,11 +48,17 @@ counter_bucket_dict = {
     'SAI_PORT_STAT_IF_OUT_OCTETS': 9
 }
 
+STATUS_NA = 'N/A'
+
 COUNTER_TABLE_PREFIX = "COUNTERS:"
 COUNTERS_PORT_NAME_MAP = "COUNTERS_PORT_NAME_MAP"
+
 PORT_STATUS_TABLE_PREFIX = "PORT_TABLE:"
 PORT_OPER_STATUS_FIELD = "oper_status"
 PORT_ADMIN_STATUS_FIELD = "admin_status"
+PORT_STATUS_UP = 'UP'
```

Comment:
This does not match the output format in previous versions. Do we want to make it consistent?

Reply:
i've updated this pull request. please check.